File: 1894-merge-strings-alternately/merge-strings-alternately.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        int i = 0, sz1 = word1.size(), sz2 = word2.size();
        string ans = "";
        while (i < sz1 && i < sz2) {
            ans = ans + word1[i] + word2[i++];
        }
        while (i < sz1) {
            ans += word1[i++];
        }
        while (i < sz2) {
            ans += word2[i++];
        }
        return ans;
    }
};

File: 1894-merge-strings-alternately/merge-strings-alternately.php
class Solution {

    /**
     * @param String $word1
     * @param String $word2
     * @return String
     */
    function mergeAlternately(string $word1, $word2) {
        $answer = ""; $i = 0 ; $j = 0;
        $sz = max(strlen($word1) , strlen($word2));
        for ($i = 0; $i < $sz; $i++){
            $answer .= ($word1[$i] ?? '').($word2[$i] ?? '');
        }
        return $answer;
    }
}